src/store/BlueprintSlice.ts

```diff
@@ -0,0 +1,53 @@
+import { PayloadAction, createSlice } from '@reduxjs/toolkit';
+
+import { RootState } from '.';
+
+type blueprintsState = {
+  selectedBlueprint: string | undefined;
+  searchInput?: string;
+  offset?: number;
```

Comment:
Would we ever want the offset and limit to be undefined? I suppose if we wanted the query to use whatever the default values are... but given that we'll be implementing pagination I think we will need fine control. Can we make these 
```
offset: number;
limit: number;
```
instead?

Reply:
The default values for offset and limit in the endpoint are `0` and `100` respectively, we can have a default values here as well, but we need to keep it consistent.

Reply:
Okay that's fine.